src/services/fs_user.c

```diff
@@ -930,7 +930,7 @@ SERVICE_CMD(0x08040000)   // GetSize
         rc = -1;
     }
 
-    DEBUG("GetSize ret=%08X(%d)\n",sz,sz);
+    DEBUG("GetSize ret=%08llX(%lld)\n",sz,sz);
```

Comment:
Try this:
#include <inttypes.h>
DEBUG("GetSize ret=%08" PRIX64 "(%" PRId64 ")\n",sz,sz);